eng/Tools.props

```diff
@@ -56,6 +56,20 @@
     <RepoTool Include="dotnet-reportgenerator-globaltool" Version="$(DotNetReportGeneratorGlobalToolPackageVersion)" />
   </ItemGroup>
 
+  <Target Name="InitOptionalTools"
```

Comment:
This is done by the YAML nuget invocation. Do you have it here to allow restore in a local build?
Would this work on CI as well?

Reply:
Right that's only for local development. The condition on it is checking an environment variable that corefx devs have set. It isn't used anymore but @safern and I decided to keep it to not introduce a breaking change in the inner dev flow.

The command can't be used by yaml without adjustments because of the `-interactive` switch. Also yaml uses nuget as dotnet restore doesn't support credential providers yet.

Reply:
I see. OK, perhaps it would be useful to add a comment to this target that documents this (including what environment variable is used, etc.)

Reply:
sounds good. we usually document internal stuff in our onenote but it makes sense to add some comments here.

Reply:
I prefer all documentation on github, except for one that would reveal secrets, so that one doesn't need to hunt for random OneNotes.